biz.aQute.tester/src/aQute/tester/plugin/ProjectTesterImpl.java

```diff
@@ -19,7 +19,8 @@
 import aQute.bnd.service.EclipseJUnitTester;
 
 public class ProjectTesterImpl extends ProjectTester implements EclipseJUnitTester {
-	int					port	= -1;
+	int					port		= -1;
+	boolean				rerunIDE	= false;
```

Comment:
I don't see where this rerunIDE instance variable is used.

Reply:
This is vestigial code and I will remove it. Its functionality has been replaced with `controlPort`, which I preferred because it could work in a backwards compatible way with `biz.aQute.tester`.

Reply:
Were you still planning on removing `rerunIDE` here?

Reply:
Fixed.